langchain/document_loaders/blob_transformers/youtube_to_audio.py

```diff
@@ -0,0 +1,33 @@
+from langchain.document_loaders.base import BaseBlobTransformer
```

Comment:
We should figure out how to organize the files. 

Here, we're dealing with classes that are called blob transformers which transform binary data to binary data of other format.

These are very similar to parsers. Parsers transform blobs to documents. Documents can be viewed as a blob that's been loaded into memory and looks like textual data.

I don't have a strong opinion at the moment, but I think we should think for a bit about the name spacing to make the code easy to discover (and relatively fast to import!)

Reply:
actually, i like your suggestion about the in memory flow.

in that case, we won't have to worry about writing these intermediate files.

i'll share that code shorty.